models/ldndc/R/met2model.LDNDC.R

```diff
@@ -0,0 +1,247 @@
+#-------------------------------------------------------------------------------
```

Comment:
For consistency and to follow rules for CRAN submissions, I think new packages should use BSD 3-Clause as is done with this package: https://github.com/PecanProject/pecan/tree/develop/base/logger